.eslintrc.js

```diff
@@ -26,5 +26,25 @@ module.exports = {
 		// do not throw an error if imported dependencies are 
 		// declared in `package.json` or `gutenberg/package.json`
 		"import/no-extraneous-dependencies": ["error", {"packageDir": ['.', './gutenberg/']}] 
-	}
+	},
+	overrides: [
+		{
+			// Workaround for addressing errors when importing react-native components.
+			// Related issue: https://git.io/JSKeJ
+			files: [ 'src/**/*.js' ],
+			settings: {
+				'import/ignore': [ 'react-native' ],
+			},
+		},
```

Comment:
This setting override was taken from Gutenberg's ESLint configuration ([reference](https://github.com/WordPress/gutenberg/blob/43d950788629daaf588c80a2a56173124f112c60/.eslintrc.js#L186-L191)).